src/log/normal.c

```diff
@@ -105,26 +105,29 @@ void normal_log_post_test(struct criterion_test_stats *stats) {
             = stats->test_status == CR_STATUS_FAILED ? CRITERION_PREFIX_FAIL :
                                              CRITERION_PREFIX_PASS;
 
-    criterion_plog(level, prefix, _(format),
-            stats->test->category,
-            stats->test->name,
-            stats->elapsed_time);
+    if(stats->test_status == CR_STATUS_SKIPPED) {
+        if(!stats->message) {
+            criterion_pinfo(CRITERION_PREFIX_SKIP, _(msg_post_test_skip),
+                    stats->test->category,
+                    stats->test->name);
+        } else {
+            criterion_pinfo(CRITERION_PREFIX_SKIP, msg_test_generic,
```

Comment:
This should be `_(msg_test_generic)` (the `_` macro calls gettext when i18n is active)